src/components/common/Extensions/TableFiltering/TableFiltering.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import { useIntl } from 'react-intl';
+import { TableFilteringProps } from './types';
+import Icon from '~shared/Icon/Icon';
+
+const displayName = 'common.Extensions.TableFiltering';
+
+const TableFiltering: FC<PropsWithChildren<TableFilteringProps>> = ({
+  filterType,
+  filterOptions,
+  className,
+  onClick,
+}) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <div
+      className={`${className} bg-blue-100 py-2 px-3 rounded-lg flex items-center gap-1`}
+    >
+      <div className="text-blue-400 font-semibold text-xs capitalize">
+        {filterType}:
+      </div>
+      {Array.isArray(filterOptions) ? (
+        filterOptions.map((name) => (
+          <p className="text-blue-400 text-4 capitalize">{name}</p>
+        ))
+      ) : (
+        <p className="text-blue-400 text-4 capitalize">{filterOptions}</p>
+      )}
+      <button
+        type="button"
+        aria-label={formatMessage({ id: 'handle.close.filter' })}
```

Comment:
I think we are adding `ariaLabel` prefix to aria-labels ->
<img width="397" alt="image" src="https://github.com/JoinColony/colonyCDapp/assets/76940796/00c9693b-71c6-4818-bab0-cedd7bc0fd39">